src/style/color.rs

```diff
@@ -225,7 +222,7 @@ impl<'de> serde::Deserialize<'de> for Color {
         }
 
         let multi_type = ColorFormat::deserialize(deserializer)
-            .map_err(|err| serde::de::Error::custom(format!("Failed to parse Colors: {err}")))?;
+            .map_err(|err| serde::de::Error::custom(format!("Failed to parse Color: {err}")))?;
```

Comment:
I don't think this can actually fail as deserializing to ColorFormat::V2 accepts any string. There's no test for this and I can't work out how to write one that triggered this message. The message in the tests is actually coming from the FromStr call, not this line).